engine/install.go

```diff
@@ -191,3 +214,26 @@ func (i *installer) errorUpdate(ctx context.Context, server *autoscaler.Server,
 	}
 	return err
 }
+
+func ParseMinuteSecondTimeDuration(str string) time.Duration {
```

Comment:
fwiw there is a duration parser in the standard library https://godoc.org/time#ParseDuration

but in this case the durations are already being parsed and set in the Config structure, so you should not need to re-parse this information.